docker/tests.Dockerfile

```diff
@@ -16,7 +16,7 @@ ENV PYTHONDONTWRITEBYTECODE=1
 COPY tests/pip_test_requirements.txt .
 RUN conda run --name hoss pip install --no-input -r pip_test_requirements.txt
 
-# Copy test directory containing Python unittest suite, test data and utilities 
+# Copy test directory containing Python unittest suite, test data and utilities
```

Comment:
Woop - the trailing whitespace check picked this up and automatically fixed it (note - that fix still needs to be added/committed, so we still get to review the magical changes)